python/generator_utils.py

```diff
@@ -86,6 +86,40 @@ def __str__(self):
 
 class MemberVariable:
   """Simple class to hold information about a member variable"""
+
+  def parse_julia_type(self, **kwargs):
```

Comment:
From a quick glance, this looks like it does not necessarily need to be a member function of the `MemberVariable` class. I would make it a free function and then simply call that on the assignment to the `self.julia_type`.

Additionally, I think it should take only one argument, or at least not a `**kwargs`, but rather named arguments instead.